src/transformers/models/gptj/modeling_gptj.py

```diff
@@ -116,8 +119,8 @@ def __init__(self, config):
         self.scale_attn = torch.sqrt(torch.tensor(self.head_dim, dtype=torch.float32)).to(torch.get_default_dtype())
 
         self.k_proj = nn.Linear(self.embed_dim, self.embed_dim, bias=False)
-        self.v_proj = nn.Linear(self.embed_dim, self.embed_dim, bias=False)
-        self.q_proj = nn.Linear(self.embed_dim, self.embed_dim, bias=False)
+        self.v_proj = LoRALinear(self.embed_dim, self.embed_dim, "selfattn", config, bias=False)
+        self.q_proj = LoRALinear(self.embed_dim, self.embed_dim, "selfattn", config, bias=False)
```

Comment:
The module instantiation of the `LoRALinear` layers have changed slightly very recently. See e.g. https://github.com/adapter-hub/adapter-transformers/blob/9eb70923ce5f40169b3176b8666d1424ded8ab3a/src/transformers/models/bert/modeling_bert.py#L270-L272


```suggestion
        self.v_proj = LoRALinear(self.embed_dim, self.embed_dim, "selfattn", config, attn_key="v", bias=False)
        self.q_proj = LoRALinear(self.embed_dim, self.embed_dim, "selfattn", config, attn_key="q", bias=False)
```